common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java

```diff
@@ -302,18 +305,15 @@ private void renderDialogue(PoseStack poseStack, List<String> currentDialogue, N
 
         @Override
         public void render(PoseStack poseStack, float partialTicks, Window window) {
-            if (NpcDialogueOverlayFeature.this.currentDialogue.isEmpty()) return;
-            if (dialogueType == NpcDialogueType.CONFIRMATIONLESS) {
-                if (System.currentTimeMillis() >= removeTime) {
-                    NpcDialogueOverlayFeature.this.currentDialogue = List.of();
-                    return;
-                }
-            }
+            if (currentDialogue.isEmpty() && confirmationlessDialogues.isEmpty()) return;
 
-            renderDialogue(
-                    poseStack,
-                    NpcDialogueOverlayFeature.this.currentDialogue,
-                    NpcDialogueOverlayFeature.this.dialogueType);
+            List<String> allDialogues = new ArrayList<>(currentDialogue);
+            confirmationlessDialogues.forEach(d -> {
+                allDialogues.add("");
```

Comment:
If there is only 1, there will be a newline?

Reply:
Yes... If you can write this without it and without making a mess out of it, please suggest. I'd need something like String.join but for a list.

Reply:
I dont think anything clearner could happen other than `if (allDialogues.Length > 0 && allDialouges[0].isBlank()) allDialogues.remove(0)`

Reply:
In fact we dont need to check list length as it is checked before

Reply:
Well that won't work. If there is a normal dialogue, and a single confirmationless dialogue, there need to be a space inbetween. The only case is if there is no normal dialogue, and just a single confirmationless dialogue.

Reply:
I fixed it anyway.